app/presenters/fully_signed_in_modal_presenter.rb

```diff
@@ -9,7 +9,22 @@ def initialize(time_left_in_session)
   def message
     t(
       'notices.timeout_warning.signed_in.message_html',
-      time_left_in_session: content_tag(:span, time_left_in_session, id: 'countdown'),
+      time_left_in_session: content_tag(:span, time_left_in_session, id: 'countdown', role: 'timer', "aria-atomic": true, ),
+    )
+  end
+
+  # set up a sc_message with role=timer, etc. create new js function to updat
+  def message
+    t(
+      'notices.timeout_warning.signed_in.message_html',
+      time_left_in_session: content_tag(:span, time_left_in_session, id: 'countdown' ),
```

Comment:
extra space
```suggestion
      time_left_in_session: content_tag(:span, time_left_in_session, id: 'countdown'),
```

Reply:
I need to get better at running the linter before opening a pr 😞